torch/csrc/jit/frontend/environment.h

```diff
@@ -1,8 +1,13 @@
+#pragma once
+
 #include <torch/csrc/jit/frontend/ir_emitter_utils.h>
```

Comment:
I don't know what the best practice™ regarding this is, but I think we should explicitly include the headers for types that are used in this file in case the includes inside `ir_emitter_utils.h` change in the future.

Reply:
Adding the extra headers leads to some `...will be initialized after [-Wreorder]...` warnings since e.g. we initialize `struct Block` in `ir.h`. I'm assuming I still shouldn't rely on transitive inclusions, right? I've changed it in the latest push

Reply:
Do we have a canonical way to suppress a warning for a line we know is fine?

Reply:
There is: https://stackoverflow.com/a/3125889

But we need some serious justification to use warning suppression I think.

What exactly is the warning you see? And what changes did you make to code to get the warning?